pages/common/blender.md

```diff
@@ -0,0 +1,32 @@
+# blender
+
+> Command-line interface to the Blender 3D computer graphics application.
+> Arguments are executed in the order they are given.
+
+- Render all frames of an animation in the background (without UI):
+
+`blender -b {{file.blend}} -a`
+
+- Render an animation to a specific output path relative to the .blend file:
+
+`blender -b {{file.blend}} -o //{{render/frame_###.png}} -a`
```

Comment:
It's to activate relative path processing (relative to the source .blend file). Without it the path is assumed to be absolute. I believe the relative option is more useful, but let me know what you think.